Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -134,4 +306,77 @@ extension PreplannedMapArea: PreplannedMapAreaProtocol {
     var description: String {
         portalItem.description
     }
+    
+    var id: PortalItem.ID? {
+        portalItem.id
+    }
+}
+
+private extension FileManager {
+    static let mmpkPathExtension: String = "mmpk"
+    static let offlineMapAreasPath: String = "OfflineMapAreas"
+    static let packageDirectoryPath: String = "Package"
+    static let preplannedDirectoryPath: String = "Preplanned"
```

Comment:
this can and should be `private` so that even within this file other types can't see them as they are implementation details:

```suggestion
    private static let mmpkPathExtension: String = "mmpk"
    private static let offlineMapAreasPath: String = "OfflineMapAreas"
    private static let packageDirectoryPath: String = "Package"
    private static let preplannedDirectoryPath: String = "Preplanned"
```